tests/model_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for the Post model."""
+
+import pytest
+from freezegun import freeze_time
+from django.utils import timezone
+from blog.models import Post
+
+
+@pytest.fixture
+def post():
+    """Creates a test Post instance."""
```

Comment:
Since this is a function and not a piece of date, the docstring should use the imperative mood.

Reply:
Marking as resolved.